### PR TITLE
Add scripts for Open-iAPS, this version for beta testers

### PR DESCRIPTION
**Pull request by marionbarker**

### CODE REVIEW

File: src/BuildOpen-iAPS.sh [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+}
+
+if [ -z "$CUSTOM_BRANCH" ]; then
+    while [ -z "$BRANCH" ]; do
+        section_separator
+        echo -e "${INFO_FONT}You are running the script to build Open-iAPS${NC}"
+        echo -e ""
+        echo -e "  ${INFO_FONT}WARNING: Beta Testers ONLY${NC}"
+        echo -e "    You should be a member of the discord server"
+        echo -e "      https://discord.gg/ptkk2Y264Z"
```
Comment (bjorkert) [inline src/BuildOpen-iAPS.sh:59]: This is the iAPS discord, not Oi?

**Review by bjorkert — CHANGES_REQUESTED** [review]

File: src/BuildOpen-iAPS.sh (hunk 1 above)
Comment (marionbarker) [reply src/BuildOpen-iAPS.sh]: updated. Thanks.

**Review by bjorkert — APPROVED** [review]

**Review by MikePlante1 — APPROVED** [review]
LGTM. Thanks!

**Comment by marionbarker** [comment]
Closing because the app was renamed.

**Pull request closed without merging**